external/build_all.sh

```diff
@@ -16,6 +16,16 @@ set -o pipefail
 
 source $(dirname "$0")/common_functions.sh
 
+if [ -n "${EXTERNAL_AQA_IMAGE}" ] ; then\
+  echo "EXTERNAL_AQA_IMAGE is decalred as $EXTERNAL_AQA_IMAGE; it would break this script. Unset it."
```

Comment:
Please see https://github.com/adoptium/aqa-tests/issues/5657 ;  I think that script shoud lbe dropped. It was broken since February. If such loop would be indeed needed, then my humble opinion is, that is should iterate through exact set of URIs, instead composing them by many IFs later in undelying scripts.

Reply:
I will verify that no one else uses the script, then ask that build_all.sh be removed as 'no use case'.

Reply:
Typo fixed. Any progress on  removing of build-all?